en/docs/includes/design/deploy-revision.md

```diff
@@ -9,11 +9,11 @@ Follow the instructions below to deploy a new revision for an API:
     
       The Deployments page appears as follows when you do not have any existing revisions.
 
-      [![New revision]({{base_path}}/assets/img/design/revision/deploy-first-revision.png){: style="width:80%"}]({{base_path}}/assets/img/design/revision/deploy-first-revision.png)
+      [![New revision](../../../../assets/img/design/revision/deploy-first-revision.png){: style="width:80%"}]({{base_path}}/assets/img/design/revision/deploy-first-revision.png)
 
       The Deployments page appears as follows when you have at least one revision.
 
-      [![Deploy new revision option](/en/4.5.0/assets/img/design/revision/deploy-new-revision.png)](/en/4.5.0/assets/img/design/revision/deploy-new-revision.png)
+      [![Deploy new revision option](../../../../assets/img/design/revision/deploy-new-revision.png)](../../../../assets/img/design/revision/deploy-new-revision.png)
```

Comment:
does this image render? I think you have to add  `{{base_path}}/assets/img/design/revision/deploy-new-revision.png`